plugins/techdocs/src/home/components/DefaultTechDocsHome.tsx

```diff
@@ -46,7 +70,13 @@ export type DefaultTechDocsHomeProps = TechDocsIndexPageProps;
  * @public
  */
 export const DefaultTechDocsHome = (props: TechDocsIndexPageProps) => {
```

Comment:
It would be nice to have some examples of filters [here](https://backstage.io/docs/next/features/techdocs/how-to-guides/#how-to-customize-the-techdocs-home-page). Could you help add them?